supervisord.conf

```diff
@@ -64,7 +64,7 @@ command =
         --homepath=/usr/share/grafana
         --config=/etc/grafana/grafana.ini
         cfg:default.paths.data=/srv/grafana
-        cfg:default.paths.plugins=/var/lib/grafana/plugins
+        cfg:default.paths.plugins=/srv/grafana/plugins
```

Comment:
Let's make sure this directory is created with `grafana:grafana` user/group

Reply:
Done.
Here is a result with the last FB

![Screenshot 2021-09-20 at 12 55 36](https://user-images.githubusercontent.com/25184101/133984640-793f8f80-cad9-4bc5-9550-387dc08dd6cd.png)

Reply:
Nice :)